src/code/code_800F9280.c

```diff
@@ -665,39 +784,43 @@ u8 func_800FAD34(void) {
     return D_80133418;
 }
 
-void func_800FADF8(void) {
-    u8 playerIdx, j;
-
-    for (playerIdx = 0; playerIdx < 4; playerIdx++) {
-        D_8016E348[playerIdx] = 0;
-        D_8016E750[playerIdx].unk_254 = NA_BGM_DISABLED;
-        D_8016E750[playerIdx].unk_256 = NA_BGM_DISABLED;
-        D_8016E750[playerIdx].unk_28 = 0;
-        D_8016E750[playerIdx].unk_18 = 0;
-        D_8016E750[playerIdx].unk_14 = 0;
-        D_8016E750[playerIdx].unk_258 = 0;
-        D_8016E750[playerIdx].unk_4D = 0;
-        D_8016E750[playerIdx].unk_4E = 0;
-        D_8016E750[playerIdx].unk_250 = 0;
-        D_8016E750[playerIdx].unk_252 = 0;
-        for (j = 0; j < 4; j++) {
-            D_8016E750[playerIdx].volScales[j] = 0x7F;
+void Audio_ResetSequences(void) {
```

Comment:
Should this be `ResetActiveSequences`, to mirror `Audio_UpdateActiveSequences`?

I assume there is another "reset sequences" function somewhere else for the audio thread sequences data

(also activeSeq -> something else eventually :tm: )